src/styles/cards.scss

```diff
@@ -0,0 +1,129 @@
+.card {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  padding: 50px 40px;
+  gap: 46px 48px;
+  justify-content: center;
+
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(1, 200px);
+}
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+ &__container {
+    display: flexbox;
+    box-sizing: border-box;
+    width: 200px;
+    height: 408px;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    font-family: $main-font-style;
+    transition: transform 300ms;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px;
+    padding-bottom: 36px;
+    margin: 0;
+  }
+
+  &__title {
+    color: $dark-color;
+    margin: 0;
+    font-size: $font-size;
+    line-height: 18px;
+    font-weight: 500;
+    padding-left: $padding-left;
+    text-align: left;
+    padding-bottom: 4px;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    color: $light-color;
+    font-size: 10px;
+    line-height: 14px;
+    padding-left: $padding-left;
+    padding-bottom: 16px;
+    margin: 0;
+  }
+
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  height: 18px;
+
+  &__title {
+    color: $light-color;
+    font-size: $font-size;
+    line-height: 18px;
+    padding-left: $padding-left;
+    margin: 0;
+  }
+
+  &__cost {
+    color: $dark-color;
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    padding-right: 16px;
+    margin: 0;
+  }
+}
```

Comment:
A good example of nesting spaces. But if you write class inside `cards.scss` it needs to be a part of `card` class. So please nest it inside `card` class and fix naming by BEM because it is a part of `card` block. And rename the file to card.scss